templates/static_templates/theme-tos.html

```diff
@@ -0,0 +1,40 @@
+## mako
+<%!
+  from django.conf import settings
+  from django.core.urlresolvers import reverse
+  from django.utils.translation import ugettext as _
+%>
+<%inherit file="/main.html" />
+
+<%namespace name='static' file='/static_content.html'/>
+
+<%block name="title"><title>${_('Terms of Service')}</title></%block>
+
+<section class="static-container tos">
+    <h2>${_('Terms of Service')}</h2>
+
+    <p>${_('Put your Terms of Service here!')}</p>
+
+    <h2 id="privacy">${_('Privacy Policy')}</h2>
+
+    <p>${_('Put your Privacy Policy here!')}</p>
+
+    <h2 id="honor">${_('Honor Code')}</h2>
+
+    <p>${_('Put your Honor Code here!')}</p>
+
+    <h2 id="copyright">${_('Copyright')}</h2>
+
+    <p>${_('Put your Copyright Text here!')}</p>
+</section>
+
+<script>
```

Comment:
yeah, will do in a future PR, this brings the default theme to parity with our other themes.